src/v/cluster/metadata_cache.cc

```diff
@@ -323,6 +323,14 @@ topic_properties metadata_cache::get_default_properties() const {
     tp.retention_local_target_ms = tristate<std::chrono::milliseconds>{
       get_default_retention_local_target_ms()};
 
+    // TODO(gellert.nagy): explicitly state all of the defaults here and add the
```

Comment:
Are you intending to make these changes in this PR or later?

Reply:
I'm planning to do this as a follow-up because I am planning to make some more changes around this that would be better as a separate PR. I am planning to remove this method altogether and fix a bug in the dry-run response of CreateTopic.